### PR TITLE
data: AbhishekSharma

### PR DESCRIPTION
**Pull request by AbhishekSharma6903**
Added My profile to LinkFree :)

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8065"></a>

### CODE REVIEW

File: data/AbhishekSharma.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/abhishek-sharma-5914bb206",
+      "icon": "FaLinkedin"
+    }
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://www.linkedin.com/in/abhishek-sharma-5914bb206",
```
Comment (kumarsonsoff3) [inline data/AbhishekSharma.json:18]: I believe it should be your GitHub URL. 

```suggestion
      "url": "https://github.com/AbhishekSharma6903",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Sone features of this app are case sensitive. So please [rename your filename here](https://github.com/AbhishekSharma6903/LinkFree/edit/AbhishekSharma6903-patch-1/data/AbhishekSharma.json) to be exact same as your GitHub username. It should be `AbhishekSharma6903.json`. 

Change `AbhishekSharma.json`  -->  `AbhishekSharma6903.json`.

Also, make the changes suggested below!

File: data/AbhishekSharma.json (hunk 1 above)
Comment (AbhishekSharma6903) [reply data/AbhishekSharma.json]: Updated the links and Name of file to AbhishekSharma6903.json

File: data/AbhishekSharma6903.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AbhiShek6903",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/abhishek-sharma-5914bb206",
+      "icon": "FaLinkedin"
+    }
```
Comment (kumarsonsoff3) [inline data/AbhishekSharma6903.json:15]: Please add the missing comma here to validate your `json`.

```suggestion
    },
```

**Review by Vishrut19 — APPROVED** [review]
Thanks for contrintuing, LGTM 👍 to me!

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**